standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2360,7 +2360,8 @@ struct GetPartitionsPsWithAuthRequest {
    9: optional i64 id=-1 // table id
    10: optional bool skipColumnSchemaForPartition,
    11: optional string includeParamKeyPattern,
-   12: optional string excludeParamKeyPattern
+   12: optional string excludeParamKeyPattern,
+   13: optional list<string> partNames;
```

Comment:
Why we need partNames in `GetPartitionsPsWithAuthRequest`? I think this request is only related to partial values, aka `ps`.